Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -32,21 +32,20 @@ public class GuessManager {
 		return guess == word
 	}
 
-	public func appendWordPlaceholder(word: String, guess: String) {
-		let fullWordGuessPlaceholder = "#"
-		if (correctlyGuessedFullWord(word, guess: guess)) {
-			correctGuesses.append(fullWordGuessPlaceholder)
+	public func appendGuess(word: String, guess: String) {
+		if (isGuessingFullWord(guess)) {
+			addGuessToCollection(correctlyGuessedFullWord, word, guess)
 		} else {
-			incorrectGuesses.append(fullWordGuessPlaceholder)
+			addGuessToCollection(hasGuessInWord, word, guess)
```

Comment:
To avoid duplication. I wanted to be able to call the same `addGuessToCollection` function for either a full word or a single letter. Otherwise I would have to split that method into two methods because `correctlyGuessedFullWord` handles a guess as a `String` and `hasGuessInWord` handles guess as a `Character`